src/pkcs15init/profile.c

```diff
@@ -2288,6 +2291,9 @@ get_authid(struct state *cur, const char *value,
 		return get_uint(cur, value, type);
 	}
 
+	if (strlen(value) >= sizeof(temp))
+		return 1;
+
```

Comment:
Note, these two issues were closed on the oss-fuzz tracker yesterday. I think these are false positives as the strlcpy should handle the bounds without this explicit check (as it has the size argument). The traces show they are using some of their reimplementation of the strlcpy instead of our internal one, which might have been the culprit. 

But having explicit check should not hurt.